package.json

```diff
@@ -11,7 +11,7 @@
     "posttest": "npm run lint",
     "test:nocov": "jest --coverage=false --forceExit",
     "test:cov": "jest --forceExit",
-    "test:watch": "jest --forceExit --watch --notify",
+    "test:watch": "jest --forceExit --coverage=fals --watch --notify",
```

Comment:
Typo? I was using this locally:

`jest --forceExit --coverageReporters html --watch --notify`

Gives a much shorter coverage summary in the jest console but also give access to a local report if you want to read it.

Reply:
I'll give that a try.  I was getting frustrated when the test fails would be pushed off the screen by the full coveragae report